optimism/src/mips/interpreter.rs

```diff
@@ -534,6 +548,30 @@ mod tests {
         };
         let exp_res = dummy_env.registers[REGISTER_AT as usize] + 27880;
         interpret_itype(&mut dummy_env, ITypeInstruction::AddImmediateUnsigned);
-        assert_eq!(dummy_env.registers[REGISTER_AT as usize], exp_res);
+        assert_eq!(
+            dummy_env.registers.general_purpose[REGISTER_AT as usize],
+            exp_res
+        );
+    }
+
+    #[test]
+    fn test_unit_jr_instruction() {
+        // We only care about instruction parts and instruction pointer
+        let mut dummy_env = dummy_env();
+        // Instruction: 0b00100100001000010110110011101000
```

Comment:
I have the feeling that this comment does not correspond to the InstructionParts below, same for the `lui at, 0xa`

Reply:
Right, old comment.